### ISSUE TITLE
Easy Vend Currency is NOT MG as set in config .

### ISSUE THREAD

**Issue by MichaelHabib** [pandorabox-io/pandorabox.io#422]
Hi,
The in game currency seems to be GoldIngot , even though the config are set to MG . Is this a bug ?
Config Line : https://github.com/pandorabox-io/pandorabox.io/blob/70a41914765e1949da3050f685a145a4b4d563e2/minetest.conf#L11

**Comment by thomasrudin** [comment]
> Is this a bug ?

Yes, this worked with the previous modified repository where the `currency` mod was specified as a dependency. Now the mod is the same as upstream and does not have this anymore so it doesn't find the `currency:minegeld` item and uses the default :confused:

**Comment by OgelGames** [comment]
> not sure how to solve that properly yet...

I think the best way to fix it, so that it works for any mod, is to delay the currency setting code to after all the mods have been loaded.

**Comment by OgelGames** [comment]
Was this fixed? I don't recall it getting fixed... 😕

**Comment by BuckarooBanzay** [comment]
> Was this fixed? I don't recall it getting fixed... confused

Not really, the issue is still there but its obviously low-prio :stuck_out_tongue: 
Feel free to reopen if you have an idea and motivation to fix that :+1:

**Comment by OgelGames** [comment]
> Feel free to reopen if you have an idea and motivation to fix that 👍

Well I know a way to fix it, as I mentioned before, I just don't know how to PR changes on that git site, I guess I could just fork it though...

Either way, the bug still exists, so the issue should be open.

**Comment by BuckarooBanzay** [comment]
> I just don't know how to PR changes on that git site

You don't :confused: last time i checked thats just a `cgit` web frontend that allows only the owner to push changes, no ability for issues or PR's..

I don't think you even expect contributions if you are putting a project on something like this :shrug:
